gulpfile.babel.js

```diff
@@ -1,63 +0,0 @@
-import gulp from 'gulp';
-import webpackStream from 'webpack-stream';
-import eslint from 'gulp-eslint';
-import flow from 'gulp-flowtype';
-import gulpFlowtype from 'gulp-flowtype';
-import { Server } from 'karma';
-import { argv } from 'yargs';
-import { WEBPACK_CONFIG_MAJOR, WEBPACK_CONFIG_MAJOR_MIN } from './webpack.conf';
-
-gulp.task('test', [ 'lint', 'karma', 'typecheck' ]);
-gulp.task('build', [ 'test', 'webpack' ]);
-
-gulp.task('webpack', [ 'webpack-major', 'webpack-major-min' ]);
-
-gulp.task('webpack-major', ['lint'], function() {
-  return gulp.src('src/index.js')
-      .pipe(webpackStream(WEBPACK_CONFIG_MAJOR))
-      .pipe(gulp.dest('dist'));
-});
-
-gulp.task('webpack-major-min', ['lint'], function() {
-  return gulp.src('src/index.js')
-      .pipe(webpackStream(WEBPACK_CONFIG_MAJOR_MIN))
-      .pipe(gulp.dest('dist'));
-});
-
-gulp.task('typecheck', [ 'lint' ], function() {
-    return gulp.src([ 'src/**/*.js', 'test/**/*.js' ])
-        .pipe(gulpFlowtype({
-            abort: true
-        }))
-});
-
-gulp.task('lint', function() {
-  return gulp.src([ 'src/**/*.js', 'test/**/*.js' ]).pipe(eslint())
-  .pipe(eslint.format())
-  .pipe(eslint.failAfterError());
-});
-
-gulp.task('karma', ['lint'], function (done) {
-
-  let server = new Server({
-    configFile: __dirname + '/karma.conf.js',
-    singleRun: !Boolean(argv['keep-browser-open']),
```

Comment:
I like having granular control over things with command-line flags. Sometimes I just want to keep the browser open and reloading as I build/fix a test, without getting all kinds of extra noisy console logs.